docs/tuning_fledge.rst

```diff
@@ -81,6 +81,61 @@ The south services within Fledge each have a set of advanced configuration optio
+    * - readLatency
+      - The longest time a reading has spent in the queue between being returned by the south plugin and sent to the storage layer.
+      - This counter describes how long, in milliseconds, the oldest reading waiting in the internal south service queue before being sent to the storage layer. This should be less than or equal to the define maximum latency, it may be a little over to allow for queue management times, but should not be significantly higher. If it is significantly higher for long periods of time it would indicate that the storage service is unable to handle the load that is being placed upon it. It may be possible that by tuning the storage layer, changing t a higher performance plugin or one that is better suited to your workload, may resolve the problem. Alternatively consider reducing the load by splitting the south services across multiple Fledge instances.
+    * - flow controlled
+      - The number of times the reading rate has been reduced due to excessive queues building up in the south service.
+      - This is closely related to the queuLength counter and has much the same set of actions that should be taken if the service is frequently flow controlled. Reducing the ingest rate, or adding filtering in the pipeline to reduce the amount of data passed onward to the storage service may alleviate the problem. In general if processing can be done that reduces high bandwidth data into lower bandwidth data that can still characterise the high bandwidth content, then this should be done as close as possible to the source of the data to reduce the overall load on the system.
+    * - throttled rate
+      - The rate that data is being ingested at as a result of flow control throttling.
+      - This counter is more for information as to what might make a reasonable ingest rate the system can sustain with the current configuration. It is useful as it gives a good idea of how far away from your desired performance the current configuration of the system is currently
+    * - storedReadings
+      - The readings successfully sent to the storage layer.
+      - This counter gives an indication of the bandwidth available from the service to the storage engine. This should be at least as high as the ingest rate if data is not to accumulate in buffers within the storage. Altering the maximum latency and maximum buffered readings advanced settings in the south server can impact this throughput.
+    * - resendQueued
+      - The number of readings queued for resend. Note that readings may be queued for resend multiple times if the resend also failed.
+      - This is a good indication of overload conditions within the storage engine. Consistent high values of this counter point to the need to improve the performance of the storage layer.
+    * - removedReadings
```

Comment:
I tried these but not able to see these counters into monitors table. Are they working?

Reply:
These are implemented inn a different pull request but have been documented here.